bedrock/legal/templates/legal/eula/thunderbird-1.5-eula.html

```diff
@@ -0,0 +1,42 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-resp.html" %}
+{% block body_class %}sand{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'legal-eula' %}
+{% endblock %}
+
+{% block page_title %}Mozilla Thunderbird End-User Software License Agreement — Version 1.5, Archived{% endblock %}
+
+{% block breadcrumbs %}
+  <nav class="breadcrumbs">
+    <a href="{{ url('mozorg.home') }}">Home</a> >
+    <a href="{{ url('legal.eula') }}">{{ _('Mozilla Foundation End-User Licensing Agreements') }}</a> >
+  </nav>
+{% endblock %}
+
+{% block content %}
+<div id="main-feature">
+  <h1 itemprop="name">Mozilla Thunderbird End-User Software License Agreement</h1>
+  <h2>Version 1.5, Archived</h2>
+</div>
+
+<div id="main-content">
+  <div class="contents">
+    <p>A SOURCE CODE VERSION OF CERTAIN THUNDERBIRD E-MAIL FUNCTIONALITY THAT YOU MAY USE, MODIFY AND DISTRIBUTE IS AVAILABLE TO YOU FREE-OF-CHARGE FROM WWW.MOZILLA.ORG UNDER THE MOZILLA PUBLIC LICENSE and other open source software licenses.</p>
+    <p>The accompanying executable code version of Mozilla Thunderbird and related documentation (the "Product") is made available to you under the terms of this MOZILLA THUNDERBIRD END-USER SOFTWARE LICENSE AGREEMENT (THE "AGREEMENT").  BY CLICKING THE "ACCEPT" BUTTON, OR BY INSTALLING OR USING THE MOZILLA THUNDERBIRD E-MAIL CLIENT, YOU ARE CONSENTING TO BE BOUND BY THE AGREEMENT.  IF YOU DO NOT AGREE TO THE TERMS AND CONDITIONS OF THIS AGREEMENT, DO NOT CLICK THE "ACCEPT" BUTTON, AND DO NOT INSTALL OR USE ANY PART OF THE MOZILLA THUNDERBIRD E-MAIL CLIENT.</p>
+    <p>DURING THE MOZILLA THUNDERBIRD INSTALLATION PROCESS, AND AT LATER TIMES, YOU MAY BE GIVEN THE OPTION OF INSTALLING ADDITIONAL COMPONENTS FROM THIRD-PARTY SOFTWARE PROVIDERS.  THE INSTALLATION AND USE OF THOSE THIRD-PARTY COMPONENTS MAY BE GOVERNED BY ADDITIONAL LICENSE AGREEMENTS.</p>
+    <p>1. LICENSE GRANT. The Mozilla Foundation grants you a non-exclusive license to use the executable code version of the Product.  This Agreement will also govern any software upgrades provided by Mozilla that replace and/or supplement the original Product, unless such upgrades are accompanied by a separate license, in which case the terms of that license will govern.</p>
```

Comment:
For consistency with [other eula pages](https://github.com/mozilla/bedrock/blob/master/bedrock/legal/templates/legal/eula/firefox-2-eula.html#L33-L43) and nicer semantics, better to change this to an ordered list?